ja_jp/contacts-search.adoc

```diff
@@ -0,0 +1,14 @@
+=== 連絡先の検索
+
+<<contacts-overview.adoc#_連絡先>> の上にある検索ボックスに様々なキーワードで連絡先を検索することが可能です。
+
+==== 連絡先の検索ボックス
+
+// .連絡先の検索
+// image::screenshots/contacts/contact-searchBox.png[連絡先の検索]
+
+メールアドレスの頭文字を入力し、image:graphics/search.svg[magnifying glass icon, width=20] のアイコンをクリックすると検索が可能です。
+
+検索するフォルダを指定する場合、検索ボックスの左側にあるドロップダウンメニューから適切のフォルダを選択します。

```

Comment:
Only two options are available; Contacts and Global Address List

Reply:
marked as resolved.
The comment has been moved to https://github.com/Zimbra/userguide/pull/10#discussion_r744458305.